apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/internal/BaseSwiftApiLiveTest.java

```diff
@@ -48,6 +48,10 @@ protected BaseSwiftApiLiveTest() {
       provider = "openstack-swift";
    }
 
+   protected void awaitConsistency() {
+      Uninterruptibles.sleepUninterruptibly(5, TimeUnit.SECONDS);
```

Comment:
Should this use the `test.blobstore.await-consistency-timeout-seconds` property as used by `BaseBlobStoreIntegrationTest`?